teacher.rb

```diff
@@ -0,0 +1,16 @@
+require_relative './person'
+
+class Teacher
+  attr_accesor :specialization
+
+  include person
```

Comment:
- Please, for readability's sake, kindly make the Teacher class inherit from the parent class Person.
`class Teacher < Person`